frontend/wallet/src/Footer.re

```diff
@@ -0,0 +1,110 @@
+open Tc;
+
+module Navigator = {
+  module Clipboard = {
+    [@bs.val] [@bs.scope ("navigator", "clipboard")]
+    external writeText: string => Js.Promise.t(unit) = "";
+
+    let writeTextTask: string => Task.t('x, unit) =
+      (str, ()) =>
+        writeText(str)
+        |> Js.Promise.then_(() => Js.Promise.resolve(Belt.Result.Ok()));
+  };
+};
```

Comment:
Yes good point

Reply:
I meant to lift the whole `navigator.clipboard`. binding up since it's a generic binding to some foreign function. Seems dirty to do foreign bindings on a per component basis.

Reply:
ah yes